src/styles/blocks/catalog.scss

```diff
@@ -12,14 +14,21 @@
     align-items: center;
     box-sizing: border-box;
 
-    width: 200px;
-    height: 300px;
+    width: $card-width;
+    height: $card-height;
     border: 1px solid #ccc;
     border-radius: 5px;
     margin-bottom: 48px;
+    transition: all $transition-duration;
+
 
     &:not(:nth-child(4n)) {
       margin-right: 48px;
     }
+
+    &:hover {
```

Comment:
It would be great to use `cursor: pointer;` for such cases. It's a good UX approach.